xhr.bs

```diff
@@ -29,6 +29,9 @@ spec:dom; type:interface; text:Document
 <pre class=anchors>
 urlPrefix: https://w3c.github.io/DOM-Parsing/; spec: dom-parsing
     type: dfn; text: fragment serializing algorithm; url: dfn-fragment-serializing-algorithm
+urlPrefix: https://wicg.github.io/feature-policy/; spec: FEATURE-POLICY
+    type: dfn; text: policy-controlled feature; url: policy-controlled-feature
+    type: dfn; text: default allowlist; url: default-allowlist
```

Comment:
These should be properly exported by Feature Policy directly. This anchors block is only there for legacy.

Reply:
They've been exported for a while -- or at least I thought they were :( Filed https://github.com/tabatkins/bikeshed/issues/1173 to get that indexed for use in bikeshed.